lib/commands/index.js

```diff
@@ -26,12 +27,30 @@ module.exports = (robot) => {
   app.post(
     /(?:un)?subscribe/,
     middleware.authenticate,
-    middleware.getResource,
+    middleware.getResource('repo'),
     middleware.getInstallation,
     middleware.pendingCommand.clear,
     subscribe,
   );
 
+  app.post(
+    '/close',
+    middleware.earlyAccess,
+    middleware.authenticate,
+    middleware.getResource(),
+    middleware.getInstallation,
+    issueState.bind(null, 'closed'),
+  );
+
+  app.post(
+    '/reopen',
+    middleware.earlyAccess,
+    middleware.authenticate,
+    middleware.getResource(),
+    middleware.getInstallation,
+    issueState.bind(null, 'open'),
```

Comment:
This is super neat, but might be confusing to folks reading it for the first time? Maybe a comment or a link explaining what this does would be good?